sites/all/modules/reol_app_feeds/src/Feed/ThemesFeed.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\reol_app_feeds\Feed;
+
+use Drupal\reol_app_feeds\Helper\NodeHelper;
+use EntityFieldQuery;
+
+/**
+ * Themes feed.
+ *
+ * @package Drupal\reol_app_feeds\Feed
+ */
+class ThemesFeed extends AbstractFeed {
+
+  /**
+   * Get themes feed data.
+   *
+   * @see https://docs.google.com/document/d/1lJ3VPAJf7DAbBWAQclRHfcltzZefUG3iGCec-z97KlA/edit?ts=5c4ef9d5#heading=h.ewzbsz8i5way
+   * for details on the feed structure.
+   *
+   * @return array
+   *   The feed data.
+   */
+  public function getData() {
+    $max_number_of_items = (int) _reol_app_feeds_variable_get('reol_app_feeds_themes', 'max_number_of_items', 50);
+
+    $entity_type = NodeHelper::ENTITY_TYPE_NODE;
+    $bundle = 'article';
+    if (node_type_load('breol_news')) {
```

Comment:
You could also use `module_exists('breol_news_page')` but the result would be the same.